src/components/molecules/Visualizer/hooks.ts

```diff
@@ -210,31 +238,47 @@ export default ({
 };
 
 function useLayers({
-  layers,
+  rootLayer,
   selected: outerSelectedPrimitiveId,
   onSelect,
 }: {
-  layers: LayerStore;
+  rootLayer: Layer | undefined;
```

Comment:
Just to keep the writing style of props in this file consistent:
```suggestion
  rootLayer?: Layer;
```